pages/docs/getting-started/event-driven-architectures.md

```diff
@@ -48,9 +48,9 @@ A message is a piece of information that's sent by the publishers to the broker,
 
 ### Channels
 
-One detail that might pass unnoticed from the diagram above is the existence of _channels_. All the _brokers_ support communication through multiple channels. The industry doesn't have a common term though, so you may find them as _topics_, _routing keys_, _event types_, etc.
+One detail that might pass unnoticed from the diagram above is the existence of channels. All brokers support communication through multiple channels. The industry doesn't have a common term for them, so you may see them referred to as _topics_, _routing keys_, _event types_, etc.
```

Comment:
```suggestion
One detail that might pass unnoticed from the diagram above is the existence of _channels_. All _brokers_ support communication through multiple channels. The industry doesn't have a common term for them, so you may see them referred to as _topics_, _routing keys_, _event types_, etc.
```

Reply:
I think we should keep `_` this same for channels and brokers otherwise everything looks good. Sentences are understandable now :)

Reply:
Yeah! Great suggestion. Thanks :)